exterminate.rb

```diff
@@ -188,7 +188,7 @@
     df.world.units.active.each do |u|
         if u.race == race_nr and checkunit[u]
             if caste_nr
-                next if caste == "enemy" && !df.unit_hostile(u)
+                next if caste == "enemy" && !df.unit_ishostile(u)
```

Comment:
Nice, I missed this (good thing I'm slow at merging PRs, I guess)

Reply:
no joke 🙈

Reply:
Not the being slow thing, just missing easy things.